autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/sync/CredentialsSync.kt

```diff
@@ -187,4 +214,22 @@ class CredentialsSync @Inject constructor(
             lastUpdatedMillis = details.lastUpdatedMillis,
         )
     }
+
+    private fun List<LoginCredentialEntry>.filterInvalidItems(): List<LoginCredentialEntry> {
+        return this.filter {
+            (it.title?.length ?: 0) < MAX_ENCRYPTED_TITLE_LENGTH &&
+                (it.domain?.length ?: 0) < MAX_ENCRYPTED_DOMAIN_LENGTH &&
+                (it.notes?.length ?: 0) < MAX_ENCRYPTED_NOTES_LENGTH &&
+                (it.username?.length ?: 0) < MAX_ENCRYPTED_USERNAME_LENGTH &&
+                (it.password?.length ?: 0) < MAX_ENCRYPTED_PASSWORD_LENGTH
+        }
+    }
+
+    companion object {
+        const val MAX_ENCRYPTED_TITLE_LENGTH = 3000
+        const val MAX_ENCRYPTED_DOMAIN_LENGTH = 1000
+        const val MAX_ENCRYPTED_USERNAME_LENGTH = 1000
+        const val MAX_ENCRYPTED_PASSWORD_LENGTH = 1000
+        const val MAX_ENCRYPTED_NOTES_LENGTH = 10000
```

Comment:
nit: can you add the thousands separator so it's easier to see it's 3,000 and not 300
- e.g., `const val MAX_ENCRYPTED_TITLE_LENGTH = 3_000`